doc/manual/expressions/language-constructs.xml

```diff
@@ -217,7 +217,25 @@ but can also be written as:
   ellipsis(<literal>...</literal>) as you can access attribute names as 
   <literal>a</literal>, using <literal>args.a</literal>, which was given as an
   additional attribute to the function.
-  </para></listitem>
+  </para>
+
+  <note>
+   <title>Caution with optional arguments</title>
+   <para>
+    As discussed in <link xlink:href="https://github.com/NixOS/nix/issues/1185">Nix#1185</link>,
+    the <literal>args@</literal> expression is bound to the argument passed to the function which means that
+    unspecified defaults won't exist in <literal>args</literal>:
```

Comment:
"unspecified defaults" is a bit vague, since the defaults *are* specified, it's the actual values that aren't.